dedibox-network/rpn/reference-content/private-networks-overview.mdx

```diff
@@ -21,7 +21,7 @@ Scaleway provides a range of solutions to help you protect against such attacks
 
 ### Understanding Private Network Types Available on the Scaleway Platform
 
-Scaleway's Public Cloud Ecosystem Scaleway Elements offers a comprehensive range of public cloud services, enabling you to build your infrastructure within the cloud. Alternatively (or in tandem), Scaleway Dedibox provides dedicated, reliable, high-end servers designed for demanding workloads. In each of these two product categories, different private network offers are available:
+Scaleway's Public Cloud Ecosystem Scaleway Elements offers a comprehensive range of public cloud services, enabling you to build your infrastructure within the cloud. Alternatively (or in tandem), Scaleway Dedibox provides dedicated, reliable, high-end servers designed for demanding workloads. In each of these two product categories, different Private Network offers are available:
```

Comment:
I think here it isn't a product name (only for Elements)

Reply:
```suggestion
Scaleway's Public Cloud Ecosystem Scaleway Elements offers a comprehensive range of public cloud services, enabling you to build your infrastructure within the cloud. Alternatively (or in tandem), Scaleway Dedibox provides dedicated, reliable, high-end servers designed for demanding workloads. In each of these two product categories, different private network offers are available:
```